src/app/layout.tsx

```diff
@@ -0,0 +1,21 @@
+import { ClerkProvider } from "@clerk/nextjs";
+
+export const metadata = {
+  title: "Oak National Academy",
+  description:
+    "Explore thousands of high-quality resources for lesson planning and curriculum design. All optional, adaptable and free.",
+};
+
+export default function RootLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  return (
+    <html lang="en">
+      <body>
+        <ClerkProvider>{children}</ClerkProvider>
+      </body>
+    </html>
+  );
+}
```

Comment:
May as well start using app router since these pages carry no baggage. This will also make it a little easier to extract these pages into a standalone authentication service later.

Reply:
should we move the onboarding pages into app too?

Reply:
@njblore I don't see why not. They have their own layout so it no pages router baggage to port across.